.github/workflows/README.md

```diff
@@ -0,0 +1,135 @@
+# Snyk
+
+We have two Snyk actions. [sbt-node-snyk](sbt-node-snyk.yml) and [sbt-node-snyk-pr](sbt-node-snyk-pr.yml).
+
+It is recommended you use both on your project for greater coverage.
+
+## Requirements
+
+1. `SNYK_TOKEN` must be available in your repo's 'Actions secrets' under 'Organization secrets' (Go to `/settings/secrets/actions`). If you cannot see this, ask an engineering manager to add it to your repo.
+1. `with: ORG:` be sure to edit this in the `yml` examples below. This will be the org within Snyk
```

Comment:
I think specifically this needs to be the organisation ID (i.e. the slug in the URL) rather than the organisation name. Often these are the same, but you can change the name and that won't change the ID. (Which has been done in the screenshot you've used)

Reply:
I'm not sure there is an org name, only an ID. The image below shows this, but I've updated it to be more specific in case they allow you to label your org in the future.